internal/provider_test/suite_test.go

```diff
@@ -606,7 +606,7 @@ type MockReconcilePolicy struct {
 func GetMockReconcilePolicy(t *testing.T) *MockReconcilePolicy {
 	cmd := exec.Command("kubectl", "get", "configmap", MOCK_OPERATOR_POLICY_CM, "-o", "jsonpath={.data}", "--ignore-not-found")
 	out, err := cmd.Output()
-	require.NoError(t, err)
+	require.NoError(t, err, "Output %s", out)
```

Comment:
The "Output " prefix will just blend into the actual output in a confusing way. Why don't you change it to something like `"OUT ===\n%s\n==="`